tsconfig.json

```diff
@@ -1,5 +1,8 @@
 {
   "compilerOptions": {
+    "lib": [
+      "es2015"
```

Comment:
Unfortunately, I think it is. The Bluebird types somewhere between 3.5.5 and 3.5.8 introduced the `Map` type which can't be handled by `tsc` with the standard settings. I added a bit of info to the commit message that added this config change: https://github.com/types/sequelize/pull/111/commits/0e75e9683ab71b1d25f9da28c543b3f32405a752